docs/sanitizers/error-alloc-dealloc-mismatch.md

```diff
@@ -1,64 +1,67 @@
 ---
 title: "Error: alloc-dealloc-mismatch"
-description: "Source examples and live debug screenshots for alloc-dealloc-mismatch errors."
-ms.date: 03/02/2021
+description: "Learn about the alloc-dealloc-mismatch Address Sanitizer error."
+ms.date: 05/28/2025
 f1_keywords: ["alloc-dealloc-mismatch"]
 helpviewer_keywords: ["alloc-dealloc-mismatch error", "AddressSanitizer error alloc-dealloc-mismatch"]
 ---
 # Error: `alloc-dealloc-mismatch`
 
 > Address Sanitizer Error: Mismatch between allocation and deallocation APIs
 
-The `alloc`/`dealloc` mismatch functionality in AddressSanitizer is off by default for Windows. To enable it, run `set ASAN_OPTIONS=alloc_dealloc_mismatch=1` before running the program. This environment variable is checked at runtime to report errors on `malloc`/`delete`, `new`/`free`, and `new`/`delete[]`.
+Enables runtime detection of mismatched memory operations that may lead to undefined behavior, such as:
+- `malloc` must be paired with `free`, not `delete`.
+- `new` must be paired with `delete`, not `free`.
+- `new[]` must be paired with `delete[]`, not `delete`.
+
+The `alloc`/`dealloc` mismatch functionality in AddressSanitizer is off by default for Windows. To enable it, run `set ASAN_OPTIONS=alloc_dealloc_mismatch=1` before running the program.
 
 ## Example
 
 ```cpp
-// example1.cpp
```

Comment:
Removal of the `example1.cpp` comment might cause confusion for the subsequent command prompt snippet and output image alt text, which reference "example 1".